lib/json_api_resource/resource.rb

```diff
@@ -35,6 +35,7 @@ def persisted?
     def save
       run_callbacks :save do
         self.client.save
+        self.meta = self.client.last_request_meta
```

Comment:
I think we had some unknown issue with that, and that is why catch_errors is an around_ thing instead of an after.  And anyway this branch should be eliminated at some point right?